README.md

```diff
@@ -240,9 +240,21 @@ player.core.activeContainer.on(Clappr.Events.CONTAINER_STATE_BUFFERING, function
 See all existing events on Clappr [here](https://github.com/clappr/clappr-core/blob/master/src/base/events.js#L227).
 
 #### plugins
-An array used to pass external plugins instances to Clappr. You can pass plugins of any category in this array.
+An object used to config external plugins instances and plugins behaviors to Clappr.
 
-Example:
+```javascript
+{
+  plugins: {
+    core: [CorePlugin],
+    container: [ContainerPlugin],
+    playback: [Playbacks],
+    disableExternalPluginsLoadPrecedence: false,
+    disableExternalPlaybacksLoadPrecedence: false,
```

Comment:
same here.

Reply:
Done on this commit 31e9f7f3dcc7980ee6889faff3bb33ce6b0e208d